static/js/components/material/Filefield_test.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import Filefield from './Filefield';
+
+describe('Filefield test', () => {
+  const renderFilefield = (props = {}) =>
+    shallow(<Filefield { ...props} />);
+
+  it('should render a link, with appropriate classes', () => {
+    const wrapper = renderFilefield();
+    assert.deepEqual(wrapper.props().className, 'button-link upload-link');
+  });
+
+  it('should stick a className on, if provided', () => {
```

Comment:
`should add a class name to the upload link if provided`

Reply:
OK